CAIPs/caip-25.md

```diff
@@ -16,422 +16,229 @@ CAIP-25 defines an authorization procedure for a chain agnostic provider to inte
+        "methods": ["eth_sendTransaction", "personal_sign"],
         "notifications": ["accountsChanged", "chainChanged"],
-        "accounts": ["eip155:1:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb", "eip155:137:0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"]
+        "capabilities": {}
       },
-      "eip155:10": {
-        "methods": ["get_balance"],
+      "eip155:8453": {
+        "accounts": ["0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"],
+        "methods": ["eth_sendTransaction", "personal_sign", "wallet_sendCalls"],
         "notifications": ["accountsChanged", "chainChanged"],
-        "accounts": []
+        "capabilities": {
+          "atomic": {
+            "status": "supported"
+          }
+        }
       },
       "eip155:42161": {
-        "methods": ["personal_sign"],
+        "accounts": ["0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"],
+        "methods": ["eth_sendTransaction", "personal_sign", "wallet_sendCalls"],
         "notifications": ["accountsChanged", "chainChanged"],
-        "accounts":["eip155:42161:0x0910e12C68d02B561a34569E1367c9AAb42bd810"],
-        "rpcDocuments": "https://example.com/wallet_extension.json"
+        "capabilities": {
+          "atomic": {
+            "status": "supported"
+          },
+          "paymasterService": {
+            "url": "https://...",
+            "optional": true
+          }
+        }
       },
-      "eip155:0": {
-        "methods": ["wallet_getPermissions", "wallet_creds_store", "wallet_creds_verify", "wallet_creds_issue", "wallet_creds_present"],
-        "notifications": []
+      "solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp": {
+        "accounts": ["7S3P4HxJpyyigGzodYwHtCxZyUQe9JiBMHyRWXArAaKv"],
+        "methods": [
+          "solana_signMessage",
+          "solana_signTransaction",
+          "solana_signAndSendTransaction"
+        ],
+        "notifications": [],
+        "capabilities": {
+          "supportedTransactionVersions": ["legacy", "0"]
+        }
       },
-      "cosmos": {
-        ...
-      }
-    },
-    "scopedProperties": {
-      "eip155:42161": {
-        "walletExtensionConfig": {
-          "foo": "bar"
+      "solana:4uhcVJyU9pJkvQyS88uRDiswHXSCkY3z": {
+        "accounts": ["6LmSRCiu3z6NCSpF19oz1pHXkYkN4jWbj9K1nVELpDkT"],
+        "methods": [
+          "solana_signMessage",
+          "solana_signTransaction",
+          "solana_signAndSendTransaction"
+        ],
+        "notifications": [],
+        "capabilities": {
+          "supportedTransactionVersions": ["legacy"]
         }
       }
     },
-    "sessionProperties": {
-      "expiry": "2022-11-31T17:07:31+00:00",
-      "globalConfig": {
-          "foo": "bar"
+    "properties": {
+      "expiry": "2022-12-24T17:07:31+00:00",
+      "walletInfo": {
+        "uuid": "350670db-19fa-4704-a166-e52e178b59d2",
+        "name": "Example Wallet",
+        "icon": "data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg'/>",
+        "rdns": "com.example.wallet"
```

Comment:
i like this over here a lot more than in the `wallet` scope

Reply:
glad to hear that